packages/devextreme-react/build.config.js

```diff
@@ -1,7 +1,7 @@
 
 module.exports = {
   src: './src/**/*.{ts,tsx}',
-  testSrc: './src/**/__tests__/*.*',
+  testSrc: './src/**/__tests__/**/*.*',
```

Comment:
Why regex here is changed? Can't find tests in your pr, that satisfy your regex version, but don't satisfy previous one